### PR TITLE
[13.0][ADD] purchase_partner_selectable_option: This module limits the partners to be selected in the purchases orders.

### PR DESCRIPTION
**Pull request by victoralmau**
This module limits the partners to be selected in the sales orders according to the **Selectable in orders** field.

The similar module for sales is: `sale_partner_selectable_option`

Please @carlosdauden and @pedrobaeza can you review it?

@Tecnativa TT33223

### CODE REVIEW

File: purchase_partner_selectable_option/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to **Contacts > Contacts**
+#. Create a record and uncheck the "Selectable in orders" box on the Purchase tab.
```
Comment (pedrobaeza) [inline purchase_partner_selectable_option/README.rst]:
```suggestion
#. Create a record and uncheck the "Selectable in orders" box on the Purchase part of the "Sales & Purchases" tab.
```

File: purchase_partner_selectable_option/README.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+#. Go to **Purchase > Orders > Request for Quotations**
+#. Create a record and the previously created partner will not appear in the Customer field.
```
Comment (pedrobaeza) [inline purchase_partner_selectable_option/README.rst]: s/Customer/Vendor?

File: purchase_partner_selectable_option/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+#. module: purchase_partner_selectable_option
+#: model:ir.model.fields,field_description:purchase_partner_selectable_option.field_res_partner__purchase_selectable
+#: model:ir.model.fields,field_description:purchase_partner_selectable_option.field_res_users__purchase_selectable
+msgid "Purchases selectable"
+msgstr "Seleccionable en compras"
+
+#. module: purchase_partner_selectable_option
+#: model_terms:ir.ui.view,arch_db:purchase_partner_selectable_option.view_partner_form
+msgid "Selectable in orders"
+msgstr "Seleccionable en presupuestos"
```
Comment (pedrobaeza) [inline purchase_partner_selectable_option/i18n/es.po]: For consistency

```suggestion
msgstr "Seleccionable en pedidos"
```

File: purchase_partner_selectable_option/models/res_partner.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    purchase_selectable = fields.Boolean(string="Purchases selectable", default=True)
```
Comment (pedrobaeza) [inline purchase_partner_selectable_option/models/res_partner.py]:
```suggestion
    purchase_selectable = fields.Boolean(string="Selectable in purchase orders", default=True)
```

**Review by pedrobaeza — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**